spec/views/users_spec.rb

```diff
@@ -0,0 +1,66 @@
+require 'rails_helper'
+require 'rails-controller-testing'
+
+RSpec.describe User, type: :system do
+  describe 'index' do
+    before do
```

Comment:
- [ ] Please add this test to user index page as the requirements ask: 
- Tests that asserts you can see the profile picture for each user.